widgetastic_manageiq.py

```diff
@@ -1761,3 +1762,109 @@ def fill(self, value):
             value = os.path.abspath(f.name)
             atexit.register(f.close)
         return super(FileInput, self).fill(value)
+
+
+class DashboardWidgetsPicker(View):
+    """Represents widgets picker in Dashboard editing screen (Cloud Intel/Reports/Dashobards)."""
+
+    ROOT = ParametrizedLocator(".//div[@id='{@id}']")
+    select = BootstrapSelect(Parameter("@select_id"))
+
+    def __init__(self, parent, id, names_locator=None, remove_locator=None,
+                 select_id=None, logger=None):
+        View.__init__(self, parent=parent, logger=logger)
+        self.id = id
+        self.names_locator = names_locator
+        self.remove_locator = remove_locator
+        self.select_id = select_id
+
+    def add_widget(self, widget):
+        self.select.fill(widget)
```

Comment:
I don't think so, here the parent widget calls `fill()` method of descendant widget.